CMakeLists.txt

```diff
@@ -37,7 +37,8 @@ endif()
 find_package(Git)
 if(GIT_FOUND AND EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/.git")
     execute_process(
-        COMMAND ${GIT_EXECUTABLE} describe --tags --always
+        COMMAND ${GIT_EXECUTABLE} describe --tags --always --match "gdxsv-*"
+        COMMAND sed "s/-\\([[:digit:]]\\{1,\\}\\)-g\\(.\\{8\\}$\\)/-dev.\\1+\\2/"
```

Comment:
Probably it doesn't work on windows.
You have to use general way to replace string. Please find those cmake command.

Reply:
Just checked, It is working in Windows.
I also tried to use `string(REGEX REPLACE ...)` command, but unfortunately it doesn't support regex quantifier `{n}`
https://stackoverflow.com/a/35845992/592511

Reply:
Thanks
Updated cmake to use `string(REGEX REPLACE ...)` command